tests/conftest.py

```diff
@@ -26,6 +27,15 @@
 )
 
 
+@pytest.fixture()
```

Comment:
Wouldn't it be better to make the fixture [autouse](https://docs.pytest.org/en/6.2.x/fixture.html#autouse-fixtures-fixtures-you-don-t-have-to-request) and avoid all the unused arguments to test functions?